agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4494,8 +4494,8 @@ bool slave_thread::handle_autoconfiguration_wsc(Socket *sd, ieee1905_1::CmduMess
                    << "     Manufacturer: " << manufacturer << std::endl
                    << "     ssid: " << ssid << std::endl
                    << "     bssid: " << network_utils::mac_to_string(bssid) << std::endl
-                   << "     authentication_type: " << authtype << std::endl
-                   << "     encryption_type: " << enctype << std::endl;
+                   << "     authentication_type: " << std::hex << int(authtype) << std::endl
```

Comment:
It would be nice if tlvf would generate a pretty-printer function `operator<<(el::base::Writer&, foo)` for every class and struct `foo` it generates. Then this would print a string instead of a hex value. Let me create a task for that: #237